prboom2/src/m_menu.c

```diff
@@ -3338,7 +3340,7 @@ setup_menu_t gen_settings2[] = { // General Settings screen2
   {"Maximum number of player corpses", S_NUM|S_PRGWARN, m_null, G_X, G_Y+12*8, {"max_player_corpse"}},
   {"Game speed, percentage of normal", S_NUM|S_PRGWARN, m_null, G_X, G_Y+13*8, {"realtic_clock_rate"}},
   {"Default skill level",              S_CHOICE,        m_null, G_X, G_Y+14*8, {"default_skill"}, 0, 0, NULL, gen_skillstrings},
-  {"Default compatibility level",      S_CHOICE,        m_null, G_X, G_Y+15*8, {"default_compatibility_level"}, 0, 0, NULL, &gen_compstrings[1]},
+  {"Default compatibility level",      S_CHOICE,        m_null, G_X-17, G_Y+15*8, {"default_compatibility_level"}, 0, 0, NULL, &gen_compstrings[1]},
```

Comment:
If you leave this line where it was and remove the extra spaces from the strings in the `gen_compstrings[]` array, which entries would exceed the line width?

Reply:
Three entries are truncated by doing this: Ultimate Doom, Boom's vanilla, and Latest PrBoom+. These could be shortened, however.

Reply:
Yes, please.

Reply:
Right, then. Could I also change Doom/2 to put the 2 in parentheses?

Reply:
Sure, why not.

Reply:
Forgot a complevel, turns out that PrBoom 2.03b gets truncated for taking up just a couple more pixels than anything else. Should PrBoom be shortened for all of them?